config.go

```diff
@@ -16,13 +16,15 @@ type Config struct {
 	PowerPin  string `toml:"power-pin"`
 	OutputDir string `toml:"output-dir"`
 	MinSecs   int    `toml:"min-secs"`
+	MaxLen    int    `toml:"max-len"`
```

Comment:
Maybe call it MaxSecs (and `max-secs`) to be consistent with the `min-secs` option? Including the unit in the name also makes the value more obvious.